README.md

```diff
@@ -1,38 +1,89 @@
 # Encryptio 
-Generic encryption software
+Encrypt strings, decrypt files and determine encryption of encrypted files. 
 
 ## Prerequisites 
-
-## Usage
+install required python in version 3.11
+```bash
+apt install python3.11
+```
 
 install required libraries via requirements 
 ```bash
 pip install -r requirements.txt
 ```
 This will download and install the [cryptography](https://pypi.org/project/cryptography/) and [cryptodome](https://pypi.org/project/pycryptodome/#description) libraries. 
 
-encryptio uses a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+## Usage of encryptio.py 
+encryptio.py use a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+### encryption
+```bash
+python3 encryptio.py encryptio.py encryption [-h] {AES,DES,3DES,IDEA,BLOWFISH} cryptoKey {CBC,CFB,CTR,ECB,OFB} cryptoData
+```
+
+Where 
+* **{AES,DES,3DES,IDEA,BLOWFISH}** - algorithm to encrypt your string with
+* **cryptoKey** - key to encrypt your string with
+* **{CBC,CFB,CTR,ECB,OFB}** - mode of encryption
+* **cryptoData** - string to encrypt
 
+### decryption
 ```bash
-python3 encryptio.py [-h] encryptionAlgorithm encryptionKey encryptionLibrary encryptionObjectIdentifier (encryptionObject)
+python3 encryptio.py decryption [-h] {AES,DES,3DES,IDEA,BLOWFISH} cryptoKey {CBC,CFB,CTR,ECB,OFB} cryptoDataFile
 ```
 
 Where 
-* **encryptionAlgorithm** - the algorithm to encrypt your object with
-* **encryptionKey** - the string key to encrypt your object with
-* **encryptionLibrary** - the encryption library to encrypt with
-* **encryptionObjectIdentifier** - the identifier of the object to encrypt
-* **encryptionObject** - the options for the object to encrypt
+* **{AES,DES,3DES,IDEA,BLOWFISH}** - algorithm to decrypt your file with
+* **cryptoKey** - key to decrypt your file with
+* **{CBC,CFB,CTR,ECB,OFB}** - mode of encryption
+* **cryptoDataFile** - file to decrypt
 
-* **-o** - the optional output file to write the encrypted object to
+## Usage of determiner.py 
+determiner.py use a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
 
-### EncryptionObjects
+### determining
+```bash
+python3 determiner.py [-h] -f encryptedFile -k encryptionKey
+```
 
-* **file** - the input file to encrypt
-* **message** - the input message to encrypt
+Where 
+* **encryptedFile** - file to analyze
+* **encryptionKey** - key to analyze 
 
 ## Examples
 
+### encrypt 
+```bash
+python3 encryptio.py encryption AES 'dasisteintestabc' CBC 'This string should be encrypted'
```

Comment:
I think as you said an english example key would be better for all examples

Reply:
PsaDqdsYnlAQjRXm is this a good random string?

Reply:
I get a exception when i try to decrypt a file with this "PsaDqdsYnlAQjRXm" 16 character key. 

`moritznentwig@MacBook-Pro-von-Moritz encryptio % python3 encryptio.py decryption AES 'PsaDqdsYnlAQjRXm' CBC encryptio_20_56_06.enc 

encryptio.py - Published by Laura Tzigiannis and Moritz Nentwig
---------------------------------------------------------------

Traceback (most recent call last):
  File "/Users/moritznentwig/dev/encryptio/encryptio.py", line 458, in <module>
    main()
  File "/Users/moritznentwig/dev/encryptio/encryptio.py", line 454, in main
    args.func(args)
  File "/Users/moritznentwig/dev/encryptio/encryptio.py", line 349, in arg_decryption_mode
    decrypt_mode(args)
  File "/Users/moritznentwig/dev/encryptio/encryptio.py", line 327, in decrypt_mode
    decrypt_aes(data, str.encode(args.cryptoKey), args.cryptoMode)
  File "/Users/moritznentwig/dev/encryptio/encryptio.py", line 203, in decrypt_aes
    plaintext = unpad(cipher.decrypt(data[2]), AES.block_size)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/Crypto/Util/Padding.py", line 92, in unpad
    raise ValueError("Padding is incorrect.")
ValueError: Padding is incorrect.`

Encrypting with this string is not a problem. Do you know whats going on?

Reply:
it works for me.. maybe you used a different mode of operation during encryption and decryption or it's the wrong file?

Reply:
U are right. Im sorry. Tested this again. 

```bash
moritznentwig@MacBook-Pro-von-Moritz encryptio % python3 encryptio.py encryption AES "PsaDqdsYnlAQjRXm" CBC "Dies ist ein teststring"

encryptio.py - Published by Laura Tzigiannis and Moritz Nentwig
---------------------------------------------------------------

Encryption successfull: File created: encryptio_18_06_57.enc
moritznentwig@MacBook-Pro-von-Moritz encryptio % python3 encryptio.py decryption AES 'PsaDqdsYnlAQjRXm' CBC encryptio_18_06_57.enc 

encryptio.py - Published by Laura Tzigiannis and Moritz Nentwig
---------------------------------------------------------------

Decryption successfull: Dies ist ein teststring
```